Origin/subscribe

milevskiyT opened the pull request.

milevskiyT committed:
Update subscribe.html

milevskiyT committed:
Update subscribe.html

milevskiyT committed:
Update subscribe.html

milevskiyT committed:
Update subscribe.html

milevskiyT committed:
Merge branch 'dev' into origin/subscribe

milevskiyT committed:
накидав моб версію

milevskiyT committed:
накидав моб версію

milevskiyT committed:
накидав моб версію

milevskiyT committed:
накидав моб версію

milevskiyT committed:
загрузив картинки + зробив таблет

milevskiyT committed:
загрузив картинки + зробив таблет

milevskiyT committed:
загрузив картинки + зробив таблет

milevskiyT committed:
рамка

milevskiyT committed:
рамка

milevskiyT committed:
рамка

milevskiyT committed:
рамка

milevskiyT committed:
і

milevskiyT committed:
і

milevskiyT committed:
Merge branch 'origin/subscribe-' of https://github.com/MaxF1996/team17-foodsta-project into origin/subscribe-

MaxF1996 committed:
Merge branch 'origin/subscribe-' of https://github.com/MaxF1996/team17-foodsta-project into origin/subscribe-

MarkYarema reviewed (commented):
Там секція називається контейнер, а дів називається  "subscribe"
Мені здається було б доцільніше поміняти місцями назви
Типу саме на дів повішати "контейнер"
На рахунок шрифтів, там мають підтягуватись з "body" шрифти щоб не вішати їх ще раз окремо
На кнопці марджин 40 стоїть вниз, тут питання таке, чи не ліпше взяти всю секцію і дати їй падінг вниз на 60 пікселів? 
Типу саме секцію розширити, не впевненний на скільки це критично, але якщо вже і робити через марджін, то задай його на форму(60px в тебе 40 стоїть)
На кнопці я не бачу стилі для тексту, там від звичайного по суті відризнається тільки font-weight ну і font-size на піксель
До фотографій додати 3х розширення
На "subscribe" флекс використовувати саме в декстопі, на таблетці вони не потрібні

milevskiyT committed:
добавив картинки 3х + шрифт на кнопку

milevskiyT committed:
добавив картинки 3х + шрифт на кнопку

milevskiyT committed:
повиправляв кольори

milevskiyT committed:
test

milevskiyT committed:
test

milevskiyT committed:
test

milevskiyT committed:
виправив помилки

milevskiyT committed:
Merge branch 'origin/subscribe-' of https://github.com/MaxF1996/team17-foodsta-project into origin/subscribe-

MaxF1996 committed:
fix

milevskiyT committed:
підігнав - марджини бананів на попередню секцію

milevskiyT committed:
Update _subscribe.scss

milevskiyT committed:
замінив назви секцій

MarkYarema commented:
Ховер і фокус кнопки задай на всі версії, не тільки для декстопу

MarkYarema commented:
font-family, font-style повидаляй з всюди в принсипі, воно наслідується з body
font-weight задай тільки там де потрібно
Додай в таблетку і декстоп на секцію subscribe- марджин ботом 100рх замість 60рх

milevskiyT committed:
зміни

milevskiyT committed:
відкорегував марджини тексту та картинки

milevskiyT committed:
---

milevskiyT committed:
зробив підпис під inputom

зробив макет підписа інпут з іконкою, сховав з display none. чекаю на допомогу колеги для підключення тексту в інпут

milevskiyT committed:
tetst

CageyOwl committed:
The alert/warning message styles were added, border styles of the input were corrected, the "required" attribute for the input was added

MaxF1996 committed:
fixed font input

milevskiyT committed:
доробив інпут ... перевір

MaxF1996 reviewed (approved):
ok

MaxF1996 committed:
Merge branch 'dev' into origin/subscribe-
